bin/build_hotfix

```diff
@@ -33,18 +38,18 @@ set -e
 pushd $IMAGE_DIR
   cmd="$CONTAINER_COMMAND build"
 
-  hotfix_images="manageiq-base manageiq-base-worker manageiq-orchestrator manageiq-webserver-worker manageiq-ui-worker"
```

Comment:
You don't want to build manageiq-base?

Reply:
Base is not shipped so it's not necessary.